### PR TITLE
Bump k8s 1.25

### PR DESCRIPTION
**Pull request by rhrazdil**

### CODE REVIEW

**Comment by rhrazdil** [comment]
check-provision-k8s-1.25 [passed](https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirtci/887/check-provision-k8s-1.25/1580892588980834304) 
Marking as ready for review

**Review by brianmcarey — COMMENTED** [review]
/lgtm

**Comment by xpivarc** [comment]
@dhiller @brianmcarey can we approve?

**Comment by rhrazdil** [comment]
Dropped the commit adding selinux labels, as it shouldn' t be required when running istio-cni pods as privileged
@brianmcarey

**Review by brianmcarey — APPROVED** [review]
Thanks to everyone involved in getting this delivered!

/approve

File: cluster-provision/k8s/1.25/istio.sh [hunk 1]
```diff
@@ -30,6 +30,7 @@ spec:
       chained: \$ISTIO_CNI_CHAINED
       cniBinDir: /opt/cni/bin
       cniConfDir: \$ISTIO_CNI_CONF_DIR
+      privileged: true
```
[the remark this answers is not in the document]
Comment (dhiller) [reply cluster-provision/k8s/1.25/istio.sh:33]: nit: wouldn't this be better stored inside a commit comment?

**Comment by dhiller** [comment]
:tada: thanks to everyone involved!

**Pull request merged**